angularjs-1.5-basics/gulp-tasks/build.js

```diff
@@ -0,0 +1,247 @@
+'use strict';
+
+var gulp = require('gulp');
+var $ = require('gulp-load-plugins')();
+var series = require('stream-series');
+var runSequence = require('run-sequence');
+var angularTemplateCache = require('gulp-angular-templatecache');
+var mergeStream = require('merge-stream');
+var plumber = require('gulp-plumber');
+var build_config = require('../config/build.config.js');
+var styleReporter = require('postcss-reporter');
```

Comment:
__[jshint]__
* 'require' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->